### PR TITLE
Issue #650: the first batch of alignments with basic wintypes

### PR DESCRIPTION
**Pull request by airelil**
Most significant changes (was in pywinauto) -> (changed as in wintypes):
BYTE: c_ubyte -> c_byte
BOOL: c_int -> c_long
HWND 32bits: c_long -> c_void_p
HWND 64bits: c_longlong -> c_void_p
HBITMAP 32bits: c_long -> c_void_p
HBITMAP 64bits: c_longlong -> c_void_p
WPARAM 32bits: c_size_t -> c_ulong
WPARAM 64bits: c_size_t -> c_ulonglong

### CODE REVIEW

**Comment by airelil** [comment]
The PR can be squashed into one commit as there are several tryouts to pass all the tests on AppVeyor

**Comment by vasily-v-ryabov** [comment]
This pull request **introduces 1 alert** when merging 96f66494aeabfad497c4b186467999ddf7f59c7e into 5418a356fab18233a9b596d68e7b1a3ce4622c3b - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-08a1b47f991f1050f3e1f7c5451c30bb7f2d3d79)

**new alerts:**

* 1 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by vasily-v-ryabov** [comment]
This pull request **fixes 1 alert** when merging 86937e0dc7cca7d9d9000445292250d0f3f7406e into 5418a356fab18233a9b596d68e7b1a3ce4622c3b - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-ad5fa2c33fc7fb2c39133a301e6d1400da51d492)

**fixed alerts:**

* 1 for Module is imported with &#39;import&#39; and &#39;import from&#39;

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: pywinauto/win32structures.py [hunk 1]
```diff
@@ -119,21 +118,21 @@ def _reduce(self):
...
-BYTE = c_ubyte
+BOOL = wintypes.BOOL
+BYTE = wintypes.BYTE
 CHAR = c_char
-DWORD = c_ulong
-HANDLE = c_void_p
-HBITMAP = c_long
-LONG = c_long
-LPVOID = c_void_p
+DWORD = wintypes.DWORD
```
Comment (vasily-v-ryabov) [inline pywinauto/win32structures.py]: I would suggest using `wintypes` directly in `win32functions` etc. So these definitions in `win32structures` would be redundant. I can agree with current change as preliminary step before we totally get rid of this re-definition.

**Comment by vasily-v-ryabov** [comment]
This pull request **fixes 1 alert** when merging 6153d4a7f4ba425479556611c66c08d17892a239 into 5418a356fab18233a9b596d68e7b1a3ce4622c3b - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-68480f766872543c7c7b5196cbd5a84cda734c50)

**fixed alerts:**

* 1 for Module is imported with &#39;import&#39; and &#39;import from&#39;

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**